doc/source/tracking/icepyx_pubs.bib

```diff
@@ -40,6 +41,47 @@ @misc{js2019agu
+year = {2021}
+}
+
+@InProceedings{10.1007/978-3-030-87013-3_13,
+author="Fernando, Garrido",
+editor="Gervasi, Osvaldo
+and Murgante, Beniamino
+and Misra, Sanjay
+and Garau, Chiara
+and Ble{\v{c}}i{\'{c}}, Ivan
+and Taniar, David
+and Apduhan, Bernady O.
+and Rocha, Ana Maria A. C.
+and Tarantino, Eufemia
+and Torre, Carmelo Maria",
+title="Mapping the Diversity of Agricultural Systems in the Cuellaje Sector, Cotacachi, Ecuador Using ATL08 for the ICESat-2 Mission and Machine Learning Techniques",
+booktitle="Computational Science and Its Applications -- ICCSA 2021",
+year="2021",
+publisher="Springer International Publishing",
+address="Cham",
+pages="170--181",
+abstract="The mapping of cropland helps to make decisions due to the intensification of its use, where the conditions of the crops change due to climatic variability and other socio-economic factors. In this way, the implementation of modern sustainable agriculture is essential to prevent soil degradation as measures to guarantee food security, propose sustainable rural development and protect the provision of different ecosystem services associated with the soil. NASA's Ice, Cloud, and Land Elevation Satellite-2 (ICESat-2) launched September 15, 2018, offers new possibilities for the mapping of global terrain and vegetation. An additional science objective is to measure vegetation canopy height as a basis for estimating large-scale biomass and biomass change. The Advanced Topographic Laser Altimeter System (ATLAS) instrument on-board ICESat-2 utilizes a photon-counting LIDAR and ancillary systems (GPS and star cameras) to measure the time a photon takes to travel from ATLAS to Earth and back again and to determine the photon's geodetic latitude and longitude. ICESat-2 ATL08 (Along-Track-Level) data product is developed for vegetation mapping with algorithms for along-track elevation profile of terrain and canopy heights retrieval of the from ATLAS point clouds. Thus, this study presents a brief look at the ATL08 product highlight the broad capability of the satellite for vegetation applications working with data of study area Seis de Julio de Cuellaje (SDJC), province of Imbabura, Ecuador. The study used Normalized Difference Vegetation Index (NDVI) by the year 2020 time-series at 30 m resolution by employing a Machine Learning (ML) approach. The results of this research indicate that the ATL08 data from the ICESat-2 product provide estimates of canopy height, show the potential for crop biomass estimation, and a machine learning land cover classification approach with a precision of 95.57{\%} with Digital Elevation Model (DEM) data.",
+isbn="978-3-030-87013-3"
+}
```

Comment:
The BibTeX for this looks a bit funny, there's no curly brackets unlike the entries above. Maybe something like this instead?
```suggestion
@incollection{FernandoMappingDiversityAgricultural2021,
  title = {Mapping the {{Diversity}} of {{Agricultural Systems}} in the {{Cuellaje Sector}}, {{Cotacachi}}, {{Ecuador Using ATL08}} for the {{ICESat-2 Mission}} and {{Machine Learning Techniques}}},
  booktitle = {Computational {{Science}} and {{Its Applications}} \textendash{} {{ICCSA}} 2021},
  author = {Fernando, Garrido},
  editor = {Gervasi, Osvaldo and Murgante, Beniamino and Misra, Sanjay and Garau, Chiara and Ble{\v c}i{\'c}, Ivan and Taniar, David and Apduhan, Bernady O. and Rocha, Ana Maria A. C. and Tarantino, Eufemia and Torre, Carmelo Maria},
  year = {2021},
  volume = {12957},
  pages = {170--181},
  publisher = {{Springer International Publishing}},
  address = {{Cham}},
  doi = {10.1007/978-3-030-87013-3_13},
  isbn = {978-3-030-87012-6 978-3-030-87013-3},
  langid = {english}
}
```